packages/library/src/brokers/rabbit/main.ts

```diff
@@ -43,18 +43,43 @@ export type RabbitListenerOptions = GenericListenerOptions & {
 
 export type RabbitMessage = GenericMessage & {};
 
-// TODO: Add error handling
+//function to act as asynchronous factory function
+/**
+ * Purpose: to produce new RabbitMQ message broker
+ * @param connection
+ * @param topics
+ * @returns RabbitMQ object to be used as a message broker
+ */
+export async function createRabbitClass(
```

Comment:
We may want to rename this to just ```Rabbit```

Reply:
Maybe we can talk through this one. Right now the "default" export is "Rabbit"

<img width="786" alt="image" src="https://user-images.githubusercontent.com/10767875/236644807-8acd25bb-ca4b-4b3c-a2df-8d1e7c1c4346.png">